libraries/SRV_Channel/SRV_Channel.h

```diff
@@ -497,14 +497,16 @@ class SRV_Channels {
     static void set_disabled_channel_mask(uint16_t mask) { disabled_mask = mask; }
 
     // add to mask of outputs which can do reverse thrust using digital controls
-    static void set_reversible_mask(uint16_t mask) {
-        reversible_mask |= mask;
-    }
+    static void set_reversible_mask(uint16_t mask);
 
     // add to mask of outputs which use digital (non-PWM) output, such as DShot
-    static void set_digital_mask(uint16_t mask) {
-        digital_mask |= mask;
-    }
+    static void set_digital_mask(uint16_t mask);
+
+    // return true if all of the outputs in mask are digital
+    static bool have_digital_outputs(uint16_t mask) { return mask && (mask & digital_mask) == mask; }
+
+    // return true if some of the outputs are digital
```

Comment:
I guess the comment is slightly incorrect in that it says the method returns true but it looks more like it returns a bitmask.

Reply:
I have changed the function so that it is truly boolean